pkg/deploy/generator/resources_dev.go

```diff
@@ -336,8 +336,8 @@ sudo -u cloud-user ./config.sh --unattended --url https://dev.azure.com/msazure
 popd
 
 # merge in /opt/rh/rh-*/enable
-cat >/home/cloud-user/agent/.path <<'EOF'
-/opt/rh/rh-python36/root/usr/bin:/opt/rh/rh-git29/root/usr/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/cloud-user/.local/bin:/home/cloud-user/bin
+cat <<EOF > /home/cloud-user/agent/.path
```

Comment:
we use `cat >fil <<EOF` in most places. Should we do it here too? Or is there good reason for other pattern, if so we should change it everywhere.

Reply:
I have changed back to that pattern now knowing it is preferred 😀